Failure in StreamEventJournalP_WmCoalescingTest.test_nonFirstPartition

http://jenkins.hazelcast.com/job/jet-oss-pr-builder/3154/testReport/junit/com.hazelcast.jet.impl.connector/StreamEventJournalP_WmCoalescingTest/test_nonFirstPartition/

Stacktrace:
```
java.lang.AssertionError: 
processor output in mode "snapshots disabled, inboxLimit=1" doesn't match , expected:<Watermark{ts=00:00:00.013}
13
Watermark{IDLE_MESSAGE}> but was:<Watermark{IDLE_MESSAGE}
Watermark{ts=00:00:00.013}
13
Watermark{IDLE_MESSAGE}>
	at com.hazelcast.jet.core.test.JetAssert.throwNotEqual(JetAssert.java:95)
	at com.hazelcast.jet.core.test.JetAssert.assertEquals(JetAssert.java:71)
	at com.hazelcast.jet.core.test.TestSupport.assertExpectedOutput(TestSupport.java:681)
	at com.hazelcast.jet.core.test.TestSupport.lambda$expectOutputs$4(TestSupport.java:338)
	at com.hazelcast.jet.core.test.TestSupport.runTest(TestSupport.java:673)
	at com.hazelcast.jet.core.test.TestSupport.assertOutput(TestSupport.java:368)
	at com.hazelcast.jet.core.test.TestSupport.expectOutputs(TestSupport.java:337)
	at com.hazelcast.jet.core.test.TestSupport.expectOutput(TestSupport.java:324)
	at com.hazelcast.jet.impl.connector.StreamEventJournalP_WmCoalescingTest.test_nonFirstPartition(StreamEventJournalP_WmCoalescingTest.java:182)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/StreamEventJournalP_WmCoalescingTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/impl/connector/StreamEventJournalP_WmCoalescingTest.java
@@ -90,7 +90,7 @@ public class StreamEventJournalP_WmCoalescingTest extends JetTestSupport {
         map.put(partitionKeys[0], 10);
         map.put(partitionKeys[1], 10);
 
-        TestSupport.verifyProcessor(createSupplier(asList(0, 1), 2000))
+        TestSupport.verifyProcessor(createSupplier(asList(0, 1), 5000))
                    .disableProgressAssertion()
                    .runUntilOutputMatches(60_000, 100)
                    .disableSnapshots()
@@ -116,7 +116,7 @@ public class StreamEventJournalP_WmCoalescingTest extends JetTestSupport {
         });
         productionStartedLatch.await();
 
-        TestSupport.verifyProcessor(createSupplier(asList(0, 1), 4000))
+        TestSupport.verifyProcessor(createSupplier(asList(0, 1), 5000))
                    .disableProgressAssertion()
                    .runUntilOutputMatches(60_000, 100)
                    .disableSnapshots()
@@ -174,7 +174,7 @@ public class StreamEventJournalP_WmCoalescingTest extends JetTestSupport {
         /* aim of this test is to check that the mapping from partitionIndex to partitionId works */
         map.put(partitionKeys[1], 13);
 
-        TestSupport.verifyProcessor(createSupplier(singletonList(1), 2000))
+        TestSupport.verifyProcessor(createSupplier(singletonList(1), 5000))
                    .disableProgressAssertion()
                    .runUntilOutputMatches(60_000, 100)
                    .disableSnapshots()

